### DISCUSSION TITLE
[QUESTION] Running raw SQL queries

### DISCUSSION THREAD

**Discussion by ghost** [jenssegers/laravel-mongodb#2218] [Q&A]
Is there any way similar to how Laravel uses the Query builder to run raw queries to run a raw SQL query into a mongodb using this package, looking something like:

```
DB::select("SELECT * FROM table")
```

**Comment by zvermafia** [comment]
I'm not sure but I think MongoDB doesn't support SQL queries so that what you want is impossible.
